### PR TITLE
Preload on create, not run

### LINKED ISSUE #44: GAS preload should be a create command option
`--preload-gas` was added to the `run` command, but it makes much more sense to add it to the `create` command

### PR DESCRIPTION
**Pull request by devhawk**
move `--preload-gas` option from `run` command to `create` command.

fixes #44

### CODE REVIEW

File: src/neo-express/Node/NodeUtility.cs [hunk 1]
```diff
@@ -172,23 +173,42 @@ void SubmitTransaction(Func<Snapshot, WalletAccount, Transaction?> factory)
...
             {
-                var generationAmount = Blockchain.GenerationAmount[0];
-                var gas = preloadGas / generationAmount;
-                return preloadGas % generationAmount == 0 ? gas : gas + 1;
+                if (i % 100 == 0)
+                {
+                    writer.WriteLine($"  Creating Block {i}");
+                }
+                var block = CreatePreloadBlock(wallet, random);
+                var relayResult = system.Blockchain.Ask<RelayResultReason>(block).Result;
```
Comment (vncoelho) [inline src/neo-express/Node/NodeUtility.cs]: Nice job, @devhawk.
Good workaround.

File: src/neo-express/BlockchainOperations.cs [hunk 2]
```diff
@@ -296,26 +297,42 @@ private static void ValidateCheckpoint(string checkPointDirectory, long magic, E
             }
         }
 
-        public static Task RunBlockchainAsync(string directory, ExpressChain chain, int index, uint secondsPerBlock, uint preloadGas, TextWriter writer, CancellationToken cancellationToken)
+        public static void PreloadGas(string directory, ExpressChain chain, int index, uint preloadGasAmount, TextWriter writer, CancellationToken cancellationToken)
         {
-            chain.InitializeProtocolSettings(secondsPerBlock);
+            Debug.Assert(preloadGasAmount > 0);
```
Comment (djnicholson) [inline src/neo-express/BlockchainOperations.cs]: it's a uint so this check probably isn't needed?

File: src/neo-express/Node/NodeUtility.cs (hunk 1 above)
Comment (devhawk) [reply src/neo-express/Node/NodeUtility.cs]: If we create all the preload blocks on a single node, wouldn't those blocks automatically sync to the other nodes automatically? 

Filed #46 to track multi-node support

Comment (vncoelho) [reply src/neo-express/Node/NodeUtility.cs]: Yes, other nodes will sync automatically, @devhawk.

Comment (devhawk) [reply src/neo-express/Node/NodeUtility.cs]: THen it's probably just a matter of updating the signing code. Will do that as part of #46

**Pull request merged**